homeassistant/components/sms/config_flow.py

```diff
@@ -79,10 +82,6 @@ async def async_step_user(self, user_input=None):
             step_id="user", data_schema=DATA_SCHEMA, errors=errors
         )
 
-    async def async_step_import(self, user_input):
-        """Handle import."""
-        return await self.async_step_user(user_input)
-
```

Comment:
Removed, as it appears to have been missed in https://github.com/home-assistant/core/pull/43484